### PR TITLE
Done. 

### PR DESCRIPTION
**Pull request by snsavage**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -3164,15 +3172,8 @@ a:visited {
...
-  padding-top: 15px;
-  margin-top: 15px;
-  border-top: 1px dotted #85929d;
-}
-
-.education-content div:nth-child(4),
-.experience-content div:nth-child(4) {
+.education-content div:nth-child(n+3),
+.experience-content div:nth-child(n+3) {
   padding-top: 15px;
```
Comment (aturkewi) [inline css/styles.css]: Hey @snsavage before I can merge this Pull Request, I'm going to need you to add all this CSS back into the sheet. These are styles that apply to all student's pages.

File: index.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<!doctype html>
-  <head>
+  <!doctype html>
+      <head>
```
Comment (aturkewi) [inline index.html]: @snsavage Can you please also change this spacing back to what it should be?

**Comment by aturkewi** [comment]
@snsavage Your pull requested looked good, but it looks like there were a few changes that you made that were out of the scope of this project. When you change those back, I'll be able to merge this pull request. Please free to reach out to me if you'd like a hand with this, you can find me on the learn-co slack as `@avidor`.

File: css/styles.css (hunk 1 above)
Comment (snsavage) [inline css/styles.css]: I was having trouble with the nth-child selector when adding experience with more than two jobs.  Changing nth-child to n+3 and removing the other numbered nth-child selector fixes that problem and will help in the future if others want more than two items under their experience or education.

**Pull request merged**